bin/s2nc.c

```diff
@@ -385,6 +403,16 @@ int main(int argc, char *const *argv)
         struct s2n_config *config = s2n_config_new();
         setup_s2n_config(config, cipher_prefs, type, &unsafe_verify_data, host, alpn_protocols, mfl_value);
 
+        if (client_cert_input != client_key_input) {
```

Comment:
should client_cert and client_key be checked too?

Reply:
I can add a notnull_check if that's what you mean after they are read from their files but the presence of the colon above ensures an argument is given

Reply:
yeah because load_file_to_cstring() can return NULL if it fails to load, this should be checked somewhere